server/channels/store/sqlstore/store.go

```diff
@@ -1289,6 +1292,34 @@ func (ss *SqlStore) toReserveCase(str string) string {
 	return fmt.Sprintf("`%s`", strings.Title(str))
 }
 
+func (ss *SqlStore) GetLocalSchemaVersion() (int, error) {
+	assets := db.Assets()
+
+	assetsList, err := assets.ReadDir(path.Join("migrations", ss.DriverName()))
+	if err != nil {
+		return 0, err
+	}
+
+	maxVersion := 0
+	for _, entry := range assetsList {
+		// parse the version name from the file name
+		m := models.Regex.FindStringSubmatch(entry.Name())
```

Comment:
nit: Ideally the migration names are sortable, so it *should* be safe to just sort `assetsList` and get the biggest value.

Reply:
Yeah, just being a bit safe here.